src/exception/exception-manifest/types.ts

```diff
@@ -62,3 +62,7 @@ export type ExceptionManifestRecord =
 	| MissingEntityExceptionRecord
 	| ProgramPropertyExceptionRecord
 	| EntityPropertyExceptionRecord;
+
+export const isEntityManifestRecord = (
+	input: ExceptionManifestRecord,
+): input is EntityPropertyExceptionRecord => input.exceptionType === 'EntityProperty';
```

Comment:
is there a constant. for `EntityProperty`? please add if it exists.

Reply:
Yes! Added